modules/pipeline-manifest/bin/_backplane_mirror_index.sh

```diff
@@ -167,8 +167,11 @@ if [[ -z $SKIP_INDEX ]]; then
   # Call make_index with mce
   make_index mce-operator-bundle $(cat .mce_operator_bundle_tag) mce-custom-registry multicluster-engine || return 1
 
-  # Call make_index with klusterlet
-  make_index klusterlet-operator-bundle $(cat .klusterlet_operator_bundle_tag) klusterlet-custom-registry rhacm2 || return 1
+  # Call make_index with klusterlet, but skip MCE 2.2 for now
+  if [[ "$PIPELINE_MANIFEST_RELEASE_VERSION" == "2.2" ]]; then echo Skip klusterlet operator bundle for now in version $PIPELINE_MANIFEST_RELEASE_VERSION;
```

Comment:
Does the echo not need to be in quotes?

Reply:
I don't think so technically. I was trying to keep with the code style of the rest of the script for uniformity and clarity